internal/controller/osko/mimirrule_controller.go

```diff
@@ -203,6 +214,15 @@ func (r *MimirRuleReconciler) newMimirClient(ds *openslov1.Datasource) error {
 	return nil
 }
 
+//func (r *MimirRuleReconciler) getMimirRuleGroupAPI(log logr.Logger, name string) (*rwrulefmt.RuleGroup, error) {
+//	mimirRule, err := r.MimirClient.GetRuleGroup(context.Background(), mimirRuleNamespace, name)
+//	if err != nil {
+//		log.Error(err, "Failed to get rule group")
+//		return nil, err
+//	}
+//	return mimirRule, nil
+//}
```

Comment:
we can probably delete this..

Reply:
this is here for the future, but yes, it's not a gamechanger